package.json

```diff
@@ -34,7 +34,8 @@
     "ensure-uploads-dir-exists": "node ./bin/ensure-uploads-dir-exists.js",
     "fontello": "gulp fontello",
     "install-dependencies": "node ./bin/install-deps.js",
-    "lint": "gulp lint",
+    "lint:watch": "esw --cache --fix --watch .",
```

Comment:
> `--fix`

I was rather thinking about adding a new separate script - `lint:fix` which would fix the linting errors on demand only.
Fixable rules coming with newer EcmaScript like `prefer-const` and `no-var` can bring havoc to our migrated files (with lots of `var`) when uncontrolled.
I'd prefer to keep `--fix` out of watch mode, or at least keep it optional.
What do you think?

Reply:
Valid point. Reverted and added documentation for how to use `--fix` manually:
```
npm run lint -- --fix
npm run lint:watch -- --fix
```

Reply:
> Cool! I didn't know one could pass additional parameters to npm like that.

I don't have many complains about NPM but with [`yarn`](https://yarnpkg.com/en/) one wouldn't need that extra clunky looking `--`  in between. 🤷‍♂️ 

>And thanks for keeping up with my whims. 🙂

Your attention to detail pleases me. ❤️